doc/usage.rst

```diff
@@ -9,9 +9,6 @@ This section describes using ``dune`` from the shell.
 Initializing Components
 =======================
 
-NOTE: The ``dune init`` command is still under development and subject to
-change.
-
```

Comment:
Why is this being removed?

Reply:
It's been there for years now, and afaik the init subcommand is not liable to any more radical changes at this point then most other parts of the CLI.

But if you all have changes planned to justify this continued earning, I'm happy to revert this part.

Reply:
I think there's indeed some plans on what to do with `dune init` in the future, so I think we should't consider this done and set in stone, especially as the post-1.0 parts of dune have rather strong backwards-compatibility guarantees.

Reply:
Ok! I've reverted this.